fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/provider/FabricTagProvider.java

```diff
@@ -233,6 +233,16 @@ private FabricTagBuilder(ObjectBuilder<T> parent) {
 			return this;
 		}
 
+		/**
+		 * Add another tag to this tag.
+		 *
+		 * @return the {@link FabricTagBuilder} instance
```

Comment:
Maybe a `@throws ClassCastException` note about `Tag.Identified` would be good here